File: src/me/krickl/memebotj/InternalCommands/UserCommands/AboutCommand.scala
package me.krickl.memebotj.InternalCommands.UserCommands

import me.krickl.memebotj._

class AboutCommand(channel: String,command: String, dbprefix: String) extends CommandHandler(channel, command, dbprefix) {
  this.setHelptext("")
  this.setUserCooldownLen(600)

	override def commandScript(sender: UserHandler, channelHandler: ChannelHandler, data: Array[String]) = {
		channelHandler.sendMessage(BuildInfo.appName + " version " + BuildInfo.version + ". Developed by " + BuildInfo.dev, this.getChannelOrigin)
		channelHandler.sendMessage("Fork me RitzMitz : https://github.com/unlink2/memebotj")
    if(Memebot.isTwitchBot) {
      channelHandler.sendMessage(f"Get me here: http://www.twitch.tv/${Memebot.mainChannel.replace("#", "")}/chat - just type !mejoin in chat! :)", this.getChannelOrigin)
    } else {
      channelHandler.sendMessage(f"Get me here: ${Memebot.mainChannel} - just type !mejoin in chat! :)", this.getChannelOrigin)
    }
	}
}

File: src/main/java/me/krickl/memebotj/Commands/Internal/AboutCommand.java
package me.krickl.memebotj.Commands.Internal;

import me.krickl.memebotj.ChannelHandler;
import me.krickl.memebotj.Commands.CommandHandler;
import me.krickl.memebotj.UserHandler;
import me.krickl.memebotj.Utility.BuildInfo;

/**
 * This file is part of memebotj.
 * Created by unlink on 11/04/16.
 */
public class AboutCommand extends CommandHandler {

    public AboutCommand(ChannelHandler channelHandler, String commandName, String dbprefix) {
        super(channelHandler, commandName, dbprefix);
    }

    @Override
    public void overrideDB() {
        this.setHelptext("");
        this.setUserCooldownLength(600);
    }

    @Override
    public void commandScript(UserHandler sender, String[] data) {
        getChannelHandler().sendMessage(BuildInfo.appName + " version " + BuildInfo.version + ". Developed by " +
                BuildInfo.dev, this.getChannelHandler().getChannel(), sender, isWhisper());
        getChannelHandler().sendMessage("Fork me RitzMitz : https://github.com/unlink2/memebotj", this.getChannelHandler().getChannel(),
                sender, isWhisper());
        getChannelHandler().sendMessage("Just type !mejoin in chat to get me! :)", this.getChannelHandler().getChannel(),
                sender, isWhisper());
    }
}
